invenio_grobid/utils.py

```diff
@@ -26,7 +26,89 @@
 
```

Comment:
Eh. This code was written in a hurry to have a functioning demo, but is code I wouldn't recommend merging: this *has* to have a better solution than the current madness.